guides/source/debugging_rails_applications.md

```diff
@@ -107,18 +107,19 @@ It can also be useful to save information to log files at runtime. Rails maintai
 
 ### What is the Logger?
 
-Rails makes use of the `ActiveSupport::Logger` class to write log information. Other loggers, such as `Log4r`, may also be substituted.
-
-You can specify an alternative logger in `config/application.rb` or any other environment file, for example:
+Rails makes use of the `ActiveSupport::Logger` class to write log information. Other loggers, such as `Log4r`, may be substituted:
 
 ```ruby
+# config/application.rb
+# or any environment file:
 config.logger = Logger.new(STDOUT)
 config.logger = Log4r::Logger.new("Application Log")
 ```
 
-Or in the `Initializer` section, add _any_ of the following
+Or:
 
-```ruby
+```rb
+# any initializer file: e.g. config/initializers/log4r.rb
 Rails.logger = Logger.new(STDOUT)
 Rails.logger = Log4r::Logger.new("Application Log")
 ```
```

Comment:
Do we even need this?

I think setting config in application / environment / initializers is covered elsewhere.

Reply:
> Do we even need this?
> I think setting config in application / environment / initializers is covered elsewhere.

Here's my reasoning:

- I made this change actually for my own benefit:
- "What is the initializer section?" I would have rather had it laid out: so I wouldn't have to even think about the answer.
- The second reason is that as a general principle, I prefer to see the the location of the markup within the code block itself, rather than in prose.

I have reverted to the old style:
```diff
- Or in the `Initializer` section, add _any_ of the following
+ Or in the `Initializer` section, add _any_ of the following:
```